src/Nethermind/Nethermind.Db.Rocks/DbOnTheRocks.cs

```diff
@@ -681,16 +691,24 @@ private static WriteOptions CreateWriteOptions(PerTableDbConfig dbConfig)
         {
             if (_readAheadReadOptions is not null && (flags & ReadFlags.HintReadAhead) != 0)
             {
-                if (!readaheadIterators.IsValueCreated)
+                Iterator iterator = iteratorManager.Rent(flags);
```

Comment:
It doesn't support `using`?

Reply:
We can make a wrapper around it. But we don't wanna always dispose the iterator.